packages/blocks/src/__internal__/utils/drag-and-drop.ts

```diff
@@ -16,7 +17,11 @@ import { type EditingState } from './types.js';
 /**
  * A dropping type.
  */
-export type DroppingType = 'none' | 'before' | 'after' | 'database';
+/**
+ * Question: Shall we add some comments here to describe each dropping type's meaning.
+ * I'd like to add a new type for drop a block into another.
+ */
+export type DroppingType = 'none' | 'before' | 'after' | 'database' | 'in';
```

Comment:
Sure, we can add some comments for these types. I think 'before' means drop blocks before target block, 'after' means drop blocks after target block, 'database' means drop blocks in database.

Reply:
So what's the difference between drop in database and not drop in database? I thought all the data are saved in database?